<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_note_window</class>
 <widget class="QWidget" name="new_note_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="new_title"/>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="new_tags_list"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
